Automattic/wp-calypso
# Store: Add expanded card view for product reviews

This PR depends on #17969. It adds the 'expanded'/open view for a product review, based off of the design from #12989.

This PR provides a 'read only' view, so the action links (approve/trash/spam) do not work yet, but they do provide the correct status. It also doesn't include replies yet.

<img width="824" alt="screen shot 2017-09-11 at 1 44 56 pm" src="https://user-images.githubusercontent.com/689165/30296471-26018dc2-96f9-11e7-8ab7-c5897ba3771c.png">

<img width="830" alt="screen shot 2017-09-11 at 1 45 12 pm" src="https://user-images.githubusercontent.com/689165/30296475-2ab5ab50-96f9-11e7-8801-6ac5ad667775.png">

Closes #17048.

To Test:
* You must use https://github.com/woocommerce/wc-api-dev/tree/add/all-product-reviews-endpoint
* Make sure you have a few reviews on your site. You can import the demo content, or post a few.
* Load up this branch.
* Go to http://calypso.localhost:3000/store/reviews/:site
* Select a review to toggle it's open state.
* Make sure that all fields show properly.
* Select a second review to verify that you can open multiple reviews at once.
* Toggle the review closed to make sure that works.
